plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/plan/CubridPlanNode.java

```diff
@@ -168,18 +200,45 @@ private void parseObject(@NotNull List<String> segments) {
         }
     }
     
+    @Nullable
+    private String getTermExtra(boolean isTerm) {
+        String[] values = terms.get(term).split(" \\(sel");
+        if(isTerm)
+            return values[0].trim();
+        else
+            return "(sel" + values[1].trim();
+    }
+
+    @Nullable
+    private String getNameOrTotal(boolean isName) {
+        Pattern p;
+        if(isName) {
+            p = Pattern.compile("\\w+ \\w+");
+        } else {
+            p = Pattern.compile("\\w+\\/\\w+");
+        }
+        Matcher m = p.matcher(classNode.get(nodeName));
+        if(m.find()) {
+            return m.group(0);
+        }
+        return null;
+    }
```

Comment:
To be honest, I'd cache these. These methods will be called every time the property is read.